Marketing.md

```diff
@@ -0,0 +1,36 @@
+# Marketing
+
+## Product Marketer Manager (Developer doing Marketing)
+
+> I am the voice of the customer, and I am the glue between engineering and marketing to drive interest and people trying our product
+
+**I have a deep knowledge of the ethos, patterns and principles of Octopus Deploy and I understand its benefits for teams and the problems it solves**
+
+* I am the authority on our customers, who are they, how they buy and why they pick Octopus (buying criteria).
+* I understand the competitive landscape, and I work to be an expert on our competition and how they are positioned.
+* I collaborate with engineering teams and marketing to develop product messaging and positioning that resonates with our customers (target personas).
+* I understand and document our customer’s buying process, including where they get information, and the who, what, when and why behind the decisions they make. Then drive changes to our sales and marketing processes based on what you learn.
```

Comment:
I help dig beneath the surface of the evidence in our marketing tools to help our marketing team understand the real buying process?

Reply:
That's a great update to the last statement which I got off a sample job description. It's a lot more readable/human.